src/Renci.SshNet/SshClient.cs

```diff
@@ -275,6 +279,30 @@ public SshCommand RunCommand(string commandText)
             return cmd;
         }
 
+#if FEATURE_TAP
+        /// <summary>
+        /// Creates and executes the command.
+        /// </summary>
+        /// <param name="commandText">The command text.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> to observe.</param>
+        /// <returns>Returns an instance of <see cref="Task{SshCommand}"/> with execution results.</returns>
+        /// <remarks>This method internally uses asynchronous calls.</remarks>
+        /// <example>
+        ///     <code source="..\..\src\Renci.SshNet.Tests\Classes\SshCommandTest.cs" region="Example SshCommand RunCommand Result Async" language="C#" title="Running simple command async" />
+        /// </example>
+        /// <exception cref="ArgumentException">CommandText property is empty.</exception>
+        /// <exception cref="T:Renci.SshNet.Common.SshException">Invalid Operation - An existing channel was used to execute this command.</exception>
+        /// <exception cref="InvalidOperationException">Asynchronous operation is already in progress.</exception>
+        /// <exception cref="SshConnectionException">Client is not connected.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="commandText"/> is <c>null</c>.</exception>
+        public async Task<SshCommand> RunCommandAsync(string commandText, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I propose to not provide a default cancellationToken. I real-life networking nothing really works all the time, and users are not aware that by not providing the CancellationToken, they are willing to wait forever. And waiting forever is almost never what they really wanted. And if somebody really wants to wait forever, he/she can still explicitly specify the default...

Reply:
Agreed. I already removed it on sshCommand.Execute but forgot for this one.